cmd/erasure-server-pool-decom.go

```diff
@@ -323,6 +346,12 @@ func (p *poolMeta) validate(pools []*erasureSets) (bool, error) {
 			continue
 		}
 		_, ok := specifiedPools[k]
+		if !ok {
```

Comment:
IIUC, we need to update the url scheme only for endpoint URLs in `rememberedPool` loaded from `pool.bin` on startup. The endpoint URLs in `specifiedPools` are based on the latest command line, so they shouldn't require any modification.

Reply:
the reason its done so to not modify what is on the disk right way, just figure out what is causing the change and verifying if the change is correct from a command line point of view. 

this is why we verify is the "Remembered" part correct even with the changes from the user.